core/src/main/java/org/kiwix/kiwixmobile/core/di/components/CoreComponent.kt

```diff
@@ -52,7 +59,8 @@ import javax.inject.Singleton
     ApplicationModule::class,
     NetworkModule::class,
     JNIModule::class,
-    DataModule::class]
+    DataModule::class,
+    ActivityModule::class]
```

Comment:
The activity module should not be here, activityModule depends on having a component created with an activity dependency provided